<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transaksi - Aplikasi Jasa Makeup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frameSidebar">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frameSidebar {
    background-color: #E91E63;
}

QPushButton {
    background-color: transparent;
    color: white;
    text-align: left;
    padding-left: 20px;
    border: none;
    font-size: 11pt;
}

QPushButton:hover {
    background-color: #C2185B;
}

QLabel {
    color: white;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblLogo">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>üíÑ MAKEUP APP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDashboard">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üè† Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPelanggan">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üë• Data Pelanggan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLayanan">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>‚ú® Layanan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnJadwal">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üìÖ Jadwal Booking</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTransaksi">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üõí Transaksi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPembayaran">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üí≥ Pembayaran</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetContent" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frameHeader">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame#frameHeader {
    background-color: white;
    border-bottom: 1px solid #ddd;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblPageTitle">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Transaksi</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblUsername">
            <property name="text">
             <string>üë§ Admin User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLogout">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
}
QPushButton:hover {
    background-color: #da190b;
}</string>
            </property>
            <property name="text">
             <string>Logout</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="pageContent" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#pageContent {
    background-color: #f5f5f5;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupInfoTransaksi">
            <property name="styleSheet">
             <string notr="true">QGroupBox {
    border: 2px solid #E91E63;
    border-radius: 8px;
    margin-top: 10px;
    padding: 15px;
    background-color: white;
    font-weight: bold;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    background-color: white;
    color: #E91E63;
}</string>
            </property>
            <property name="title">
             <string>‚ÑπÔ∏è Informasi Transaksi</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Pelanggan:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cmbPelanggan">
               <property name="styleSheet">
                <string notr="true">QComboBox {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Jadwal:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QComboBox" name="cmbJadwal">
               <property name="styleSheet">
                <string notr="true">QComboBox {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Tanggal:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QDateEdit" name="dateTanggal">
               <property name="styleSheet">
                <string notr="true">QDateEdit {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupDetailLayanan">
            <property name="styleSheet">
             <string notr="true">QGroupBox {
    border: 2px solid #E91E63;
    border-radius: 8px;
    margin-top: 10px;
    padding: 15px;
    background-color: white;
    font-weight: bold;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    background-color: white;
    color: #E91E63;
}</string>
            </property>
            <property name="title">
             <string>üõçÔ∏è Detail Layanan</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QFrame" name="frame">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="leftMargin">
                 <number>11</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Item Layanan</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="btnTambahLayanan">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    background-color: #E91E63;
    color: white; 
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #C2185B;
}</string>
                  </property>
                  <property name="text">
                   <string>‚ûï Tambah Layanan</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="tableDetailLayanan">
               <property name="styleSheet">
                <string notr="true">QTableWidget {
    background-color: white;
    border: 1px solid #ddd;
    gridline-color: #f0f0f0;
}
QTableWidget::item:selected {
    background-color: #FCE4EC;
    color: #333;
}
QHeaderView::section {
    background-color: #E91E63;
    color: white;
    padding: 8px;
    border: none;
    font-weight: bold;
}</string>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string>Layanan</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Harga</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Jumlah</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Subtotal</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Aksi</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frameTotalTransaksi">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QGroupBox" name="groupTotal">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
    border: 2px solid #E91E63;
    border-radius: 8px;
    margin-top: 8px;
    background-color: white;
    font-weight: bold;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    background-color: white;
    color: #E91E63;
}</string>
               </property>
               <property name="title">
                <string>üí∞ Total</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <property name="leftMargin">
                 <number>15</number>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>15</number>
                </property>
                <property name="bottomMargin">
                 <number>10</number>
                </property>
                <item row="0" column="1">
                 <widget class="QLabel" name="lblSubtotal">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Rp 0</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Diskon:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>GRAND TOTAL:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="lblGrandTotal">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #E91E63;</string>
                  </property>
                  <property name="text">
                   <string>Rp 0</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="lblDiskon">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Rp 0</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Subtotal:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnSimpanTransaksi">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    border-radius: 5px;
    font-weight: bold;
    font-size: 11pt;
}
QPushButton:hover {
    background-color: #45a049;
}</string>
               </property>
               <property name="text">
                <string>üíæ Simpan Transaksi</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnBatalTransaksi">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #da190b;
}</string>
               </property>
               <property name="text">
                <string>‚ùå Batal</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupHistory">
            <property name="styleSheet">
             <string notr="true">QGroupBox {
    border: 2px solid #E91E63;
    border-radius: 8px;
    margin-top: 10px;
    padding: 15px;
    background-color: white;
    font-weight: bold;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    background-color: white;
    color: #E91E63;
}</string>
            </property>
            <property name="title">
             <string>üìã Riwayat Transaksi</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QTableWidget" name="tableHistoryTransaksi">
               <property name="styleSheet">
                <string notr="true">QTableWidget {
    background-color: white;
    border: 1px solid #ddd;
    gridline-color: #f0f0f0;
}
QTableWidget::item:selected {
    background-color: #FCE4EC;
    color: #333;
}
QHeaderView::section {
    background-color: #E91E63;
    color: white;
    padding: 8px;
    border: none;
    font-weight: bold;
}</string>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string>ID Transaksi</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Tanggal</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Pelanggan</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Total</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Status</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Aksi</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
